docs/user-guide/pod-states.md

```diff
@@ -5,6 +5,25 @@ Updated: 4/14/2015
 
 This document covers the lifecycle of a pod.  It is not an exhaustive document, but an introduction to the topic.
```

Comment:
@johndmulhausen Is there a standard way of documenting when the doc was last updated? (see line 4)